File: app/src/main/java/com/rachelleignacio/listbucket/domain/interactors/CreateListInteractor.java
package com.rachelleignacio.listbucket.domain.interactors;

import com.rachelleignacio.listbucket.domain.interactors.base.Interactor;
import com.rachelleignacio.listbucket.domain.models.List;

/**
 * Created by rachelleignacio on 3/7/17.
 */

public interface CreateListInteractor extends Interactor {
    interface Callback {
        void onListCreated(List newList);
    }
}

File: app/src/main/kotlin/com/rachelleignacio/listbucket/domain/interactors/CreateListInteractor.kt
package com.rachelleignacio.listbucket.domain.interactors

import com.rachelleignacio.listbucket.domain.interactors.base.Interactor
import com.rachelleignacio.listbucket.domain.models.List

/**
 * Created by rachelleignacio on 8/9/17.
 */
interface CreateListInteractor : Interactor {
    interface Callback {
        fun onListCreated(newList: List)
    }
}